firmware/application/apps/ui_mictx.cpp

```diff
@@ -355,9 +355,9 @@ MicTXView::MicTXView(
                 rxaudio(rx_enabled);         // Update now if we have RX audio on
                 options_tone_key.hidden(0);  // we are in FM mode, we should have active the Key-tones & CTCSS option.
 
-                rxbw.emplace_back(" NFM1:8k5  ", 0);  // restore the original dynamic field_rxbw value.
-                rxbw.emplace_back(" NFM2:11k  ", 1);
-                rxbw.emplace_back(" FM  :16k  ", 2);
+                rxbw.emplace_back(" 8k5  ", 0);  // restore the original dynamic field_rxbw value.
```

Comment:
To keep in sync with naming conventions, you might want to consider replacing these 4 lines with:  freqman_set_bandwidth_option(NFM_MODULATION, field_rxbw);

Reply:
Hi,  thanks for the tick,  I did not know that this compact function can be used when we need to refresh dynamically those sub options.  Let me try and check , and if all ok, I will update the PR.

Reply:
Maybe we are missing something more , ... when applying it ,  now the filed  is  totally black and not possible to be selected, 

freqman_set_bandwidth_option(NFM_MODULATION, field_rxbw); // adding the common message from freqman.cpp 
                // rxbw.emplace_back(" 8k5  ", 0);  // restore the original dynamic field_rxbw value.
                // rxbw.emplace_back(" 11k  ", 1);
                // rxbw.emplace_back(" 16k  ", 2);
field_rxbw.set_options(rxbw);

is that what you mean ?

Reply:
You need to delete the "field_rxbw.set_options(rxbw);" line as well.

Reply:
Ahhh  I see,  , also commenting the last 4th line as you said, and now it works fine !!!!
thanks,
Tomorrow I will  ammend the PR ..

Reply:
This is solved ,